README.md

```diff
@@ -35,283 +35,22 @@ and other user-facing objects are subject to change. Backward-compability is not
-
-Start mysql and wordpress:
-
-```
-kubectl create -f mysql.yaml
-kubectl create -f wordpress.yaml
-```
-
-Both of these apps create a block volume and mount it to their respective pod. You can see the Kubernetes volume claims by running the following:
-
-```
-$ kubectl get pvc
-NAME             STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
-mysql-pv-claim   Bound     pvc-95402dbc-efc0-11e6-bc9a-0cc47a3459ee   20Gi       RWO           1m
-wp-pv-claim      Bound     pvc-39e43169-efc1-11e6-bc9a-0cc47a3459ee   20Gi       RWO           1m
-```
-
-Once the wordpress and mysql pods are in the `Running` state, get the cluster IP of the wordpress app and enter it in your brower:
-
-```
-$ kubectl get svc wordpress
-NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
-wordpress   10.3.0.155   <pending>     80:30841/TCP   2m
-```
-
-You should see the wordpress app running.  
-
-**NOTE:** When running in a vagrant environment, there will be no external IP address to reach wordpress with.  You will only be able to reach wordpress via the `CLUSTER-IP` from inside the Kubernetes cluster.
-
-#### Rook Client
-You also have the option to use the `rook` client tool directly by running it in a pod that can be started in the cluster with:
-```
-kubectl create -f rook-client/rook-client.yml
-```  
-
-Starting the rook-client pod will take a bit of time to download the container, so you can check to see when it's ready with (it should be in the `Running` state):
-```
-kubectl -n rook get pod rook-client
-```
-
-Connect to the rook-client pod and verify the `rook` client can talk to the cluster:
-```
-kubectl -n rook exec rook-client -it bash
-rook node ls
-```
-
-At this point (optional), you can follow the steps in the [Block, File and Object Storage section](#block-file-and-object-storage) to create and use those types of storage.
-
-## Rook Standalone Service
-
-Rook can also be deployed as a standalone service on any modern Linux host by running the following:
-
-### Linux
-1. Download the latest  binaries
-
-    ```bash
-    $ wget https://github.com/rook/rook/releases/download/v0.2.2/rook-v0.2.2-linux-amd64.tar.gz
-    $ tar xvf rook-v0.2.2-linux-amd64.tar.gz
-    ```
-
-2. Start a one node Rook cluster
-
-    ```bash
-    $ ./rookd --data-dir /tmp/rook-test
-    ```
-
-### Vagrant
-
-Rook is also easy to run with `vagrant` on CoreOS via `rkt`.
-
-```
-cd demo/vagrant
-vagrant up
-```
+## Quickstart Guides
 
-## Block, File and Object Storage
+There are a few different options for running a Rook cluster for your storage needs.  Kubernetes is the recommended way because of the rich orchestration and scheduling that Kubernetes provides via the Rook operator.
 
-### Block Storage
-1. Create a new volume image (10MB)
+1. [Kubernetes](/demo/kubernetes/README.md) (recommended)
+2. [Rook Standalone Service](/demo/standalone/README.md)
+3. [Vagrant](/demo/vagrant/README.md)
```

Comment:
"Vagrant" is just another way to run the Standalone. Seems like we should move this option to the Standalone page.